Xamarin.Forms.Controls/CoreGallery.cs

```diff
@@ -573,6 +573,7 @@ void ValidateRegistrar()
 
 			foreach (var page in Issues.Helpers.ViewHelper.GetAllPages())
 			{
+				page.Visual = VisualMarker.Default;
```

Comment:
This ensures that these all run with visual set to default. If we make it so Material is default then this test code here will cause all the nonappcompat renderers to register against Material. Setting this to Default makes sure everything wires up in the registrar correctly